packages/universal-swap/src/handler.ts

```diff
@@ -562,7 +562,7 @@ export class UniversalSwapHandler {
     try {
       const _fromAmount = toAmount(fromAmount, fromTokenOnOrai.decimals).toString();
 
-      if (!this.swapData.simulatePrice || !this.swapData.userSlippage)
+      if (!this.swapData.simulatePrice || !(this.swapData.userSlippage || this.swapData.userSlippage === 0))
```

Comment:
We should not group condition like this. If its all || then its more readable to remove the ()

Reply:
Also, we need an extra test case for that new condition

Reply:
Also, I'd like to have this condition wrapped in a variable to provide context to the condition.

Reply:
const isValidSlippage = this.swapData.userSlippage || this.swapData.userSlippage === 0;
if (!this.swapData.simulatePrice || !isValidSlippage)